docs/source/_static/style.css

```diff
@@ -1,11 +1,4 @@
 div.graphviz {
     overflow: auto;
-    max-width: 620px;
-    max-height: 50px;
-    text-align: center;
+    text-align: left;
```

Comment:
Weird, so `graphviz` isn't respecting this.

Reply:
@dsherry Yea, when I look at the html and css styling of the rendered page, this `graphviz` element is a larger container that contains the rendered image, so any text-aligning done here won't have an impact on the image contents.